src/api/app/views/person/token/index.xml.builder

```diff
@@ -1,6 +1,6 @@
 xml.directory(count: @list.length) do |dir|
   @list.each do |token|
-    p = { id: token.id, string: token.string, kind: token.token_name, name: token.name, triggered_at: token.triggered_at }
+    p = { id: token.id, string: token.string, kind: token.token_name, name: token.description, triggered_at: token.triggered_at }
```

Comment:
Hm I think we can rename this too. There is no `name` support in `osc`, and we introduced `name` just 6 months ago (b3f0122ca0866a866de282b73d36f9b1f6a11ebb)...

Reply:
ah just 6 months ago? Yes for sure in such short period time, the osc team didn't pick up this change. I will change it.

Reply:
fixed